src/lib/make_svg.rs

```diff
@@ -313,6 +313,9 @@ fn get_move_numbers_to_draw(goban: &Goban, options: &MakeSvgOptions) -> Vec<(u64
 }
 
 fn get_move_numbers(goban: &Goban, options: &MakeSvgOptions) -> Vec<(u64, Stone)> {
+    if !options.draw_move_numbers && !options.kifu_mode {
```

Comment:
Could you remove the `&& !options.kifu_mode` here? `draw_move_numbers` gets set automatically for `kifu_mode` so the check is redundant.